### ISSUE TITLE
We should have the new ELI5 Velox video from youtube embedded in the homepage

### ISSUE THREAD

**Issue by psbell-meta** [facebookincubator/velox#4245]
### Description

We should replace the current explainer diagram with [this video](https://www.youtube.com/watch?v=T9NMWN7vuSc&list=PLzIwronG0sE49zLk608yB5GKKT8WbEhnZ)

**Comment by kgpai** [comment]
@psbell-meta Can you create an image for it ? 
I dont think we can embed videos in mark down pages at the moment : https://stackoverflow.com/questions/11804820/how-can-i-embed-a-youtube-video-on-github-wiki-pages

**Comment by psbell-meta** [comment]
@kgpai PR is already approved, I tested local and loaded the staging view and the embed worked fine, I think Docusaurus handles it
https://deploy-preview-4244--meta-velox.netlify.app/

Let me know if you still have concerns

**Comment by kgpai** [comment]
I am sorry I thought you meant the Velox github README page (https://github.com/facebookincubator/velox#readme)  and not netlify page. No concerns at all from my end; I misunderstood what the home page was .
